Resources/public/templates/fields/view/binaryfile.hbt

```diff
@@ -4,7 +4,7 @@
     </div>
     <div class="ez-fieldview-value pure-u"><div class="ez-fieldview-value-content">
     {{#if isEmpty}}
-        This field is empty
+        {{translate 'fieldview.field.empty' 'fieldview'}}
     {{else}}
         <a href="{{ value.url }}" title="Download the file">{{ value.fileName }}</a> ({{ value.fileSize }}&nbsp;bytes)
```

Comment:
also missing for "Download the file" title. On the *bytes*, we should probably keep it this way until we have strings with parameters in place.